demos/python_demos/object_detection_demo_ssd_async/object_detection_demo_ssd_async.py

```diff
@@ -50,6 +50,51 @@ def build_argparser():
     return parser
 
 
+class SingleOutputPostprocessor:
+    def __init__(self, output_layer):
+        self.output_layer = output_layer
+
+    def __call__(self, outputs):
+        return outputs[self.output_layer][0][0]
+
+
+class MultipleOutputPostprocessor:
+    def __init__(self, bboxes_layer='bboxes', scores_layer='scores', labels_layer='labels'):
+        self.bboxes_layer = bboxes_layer
+        self.scores_layer = scores_layer
+        self.labels_layer = labels_layer
+
+    def __call__(self, outputs):
+        bboxes = outputs[self.bboxes_layer][0]
+        scores = outputs[self.scores_layer][0]
+        labels = outputs[self.labels_layer][0]
+        return [[0, label, score, *bbox] for label, score, bbox in zip(labels, scores, bboxes)]
+
+
+def get_output_postprocessor(net, bboxes='bboxes', labels='labels', scores='scores'):
+    if len(net.outputs) == 1:
+        output_blob = next(iter(net.outputs))
+        return SingleOutputPostprocessor(output_blob)
+    elif len(net.outputs) >= 3:
+        def find_layer(name, output_name, all_outputs):
```

Comment:
`output_name` is used only to report an error. Looks this argument should be removed and the error messages should be based on `name`.